### DISCUSSION TITLE
在线访问连接时间可控

### DISCUSSION THREAD

**Discussion by 952362** [AlistGo/alist#6352] [Ideas]
在挂载 115 网盘之后，我向其存储了一些大存储的视频电影，其中基本超过 86gb，在正常访问alist时，无法观看，我认为可能是连接时间过短，能不能加一个选项用来调控
![Screenshot_2024-04-15-13-17-35-940_com yujian ResideMenuDemo](https://github.com/alist-org/alist/assets/140718382/f3e707f2-d01d-48c3-ad44-0c39406f978a)
还有，我的浏览器支持查看iso格式文件，只要小一点都可以看

**Comment by pongfcnkl** [comment]
开本地代理 用本地播放器加webdav观看

**Reply by 952362** [reply]
不知道是不是这样做，但效果好像没有
![Screenshot_2024-04-15-20-26-12-449_com yujian ResideMenuDemo](https://github.com/alist-org/alist/assets/140718382/de5f9c1e-a6f0-4473-93f8-dcb6cfaf852d)
![Screenshot_2024-04-15-20-26-01-084_com yujian ResideMenuDemo](https://github.com/alist-org/alist/assets/140718382/5a75bee7-cd9c-4b44-9042-c94b09e3c512)
我是搭建在云服务器上的，开启本地代理我觉得可能会导致更糟糕的局面

**Reply by pongfcnkl** [reply]
@952362 
你这是安卓手机，可以不用服务器直接搭建在手机上
